README.adoc

```diff
@@ -0,0 +1,75 @@
+[[minishift-addons]]
+= Minishift Add-ons
+:icons:
+:toc: macro
+:toc-title:
+:toclevels: 1
+
+toc::[]
+
+[[welcome-to-minishift]]
+== Welcome to the Minishift add-ons repository!
+
+This repository contains add-ons contributed by the Minishift community.
+
+We hope that by sharing these add-ons, folks in the community will be able to make the most out of their Minishift environment.
+Community add-ons are tested and supported by the community.
+
+[[introduction-addons]]
+== Introduction
+
+link:https://docs.openshift.org/latest/minishift/using/addons.html[Minishift add-ons] are extensions, or plug-ins, that customize the default OpenShift cluster configuration and behavior.
+For example, you can use the link:https://github.com/minishift/minishift/tree/master/addons/anyuid[anyuid] add-on to allow authenticated users to run images without a pre-allocated UID, or the link:https://github.com/minishift/minishift/tree/master/addons/registry-route[registry-route] add-on to expose the OpenShift registry as a route.
+
+Minishift includes several link:https://docs.openshift.org/latest/minishift/using/addons.html#default-addons[default add-ons], which are stored in *_$MINISHIFT_HOME/addons_*.
```

Comment:
See this is confusing. Minishift provides indeed a set of default add-ons, but the fact that they are stored in _$MINISHIFT_HOME/addons_ is not specific to these add-ons. All add-ons are installed there. 

"Minishift includes several default add-ons which are installed automatically during the first start of Minishift. These add-ons are tested and supported by the Minishift core team."